library/common.h

```diff
@@ -29,6 +29,7 @@
 #include "mbedtls/config.h"
 #endif
 
+#if defined(MBEDTLS_TEST_HOOKS)
```

Comment:
The file would be more readable if it was structured by declaration than by compilation option. Also, the documentation here applies whether or not `MBEDTLS_TEST_HOOKS` is enabled. So please keep the structure the way it was before.

```
/** documentation of MBEDTLS_STATIC_TESTABLE */
#if defined(MBEDTLS_TEST_HOOKS)
#define MBEDTLS_STATIC_TESTABLE
#else
#define MBEDTLS_STATIC_TESTABLE static
#endif

/** documentation of MBEDTLS_ERR_ADD */
#if defined(MBEDTLS_TEST_HOOKS)
#define MBEDTLS_ERR_ADD mbedtls_err_add()
void mbedtls_err_add();
#else
#define MBEDTLS_ERR_ADD (high+low)
#endif
```

Reply:
Thanks, I wasn't sure of myself when I made the change originally. I will make the change now.

Reply:
Done.